controllers/admin/AdminSearchConfController.php

```diff
@@ -181,6 +181,49 @@ public function __construct()
                             ),
                         ),
                     ),
+                    'PS_SEARCH_FUZZY' => array(
+                        'title' => $this->trans('Fuzzy Search', array(), 'Admin.Shopparameters.Feature'),
+                        'validation' => 'isBool',
+                        'cast' => 'intval',
+                        'type' => 'bool',
+                        'desc' => $this->trans(
+                                'By default, fuzzy search is enabled and spelling error are allowed. To search “bird”, you can to enter “burd”, “bard”, “beerd” etc...',
+                                array(),
+                                'Admin.Shopparameters.Help'
+                            ) . '<br/>' .
+                            $this->trans(
+                                'With this option disabled, to get good result, you need get exact writting contained in the word.',
+                                array(),
+                                'Admin.Shopparameters.Help'
+                            ),
+                        'hint' => array(
+                            $this->trans(
+                                'Enable spelling errors during search.',
```

Comment:
How about _Enable approximate string matching_? Is it a feature or a help text?

Reply:
Would it be a feature or a help text?

Reply:
1. I forgot to reply, but already done.
2. Both